src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -258,7 +258,7 @@ SGVector<T> from_gpu(const SGVector<T>& vec)
 			SG_SERROR("Data memory on GPU but no GPU backend registered. \
 						This can happen if the GPU backend was de-activated \
 						after memory has been transferred to GPU.\n");
-			return NULL;
+			return false;
```

Comment:
I did this because I got warning when I used "return NULL":
```
/Users/zora/Github/shogun/src/shogun/mathematics/linalg/LinalgNamespace.h:338:11: warning: implicit conversion of NULL constant to 'bool' [-Wnull-conversion]
   return __null;
          ^~~~~~
          false
```
Though I do think NULL makes more sense..
Or we can leave the warning there @vigsterkr